assets/js/components/audit_trails/AuditTable.jsx

```diff
@@ -20,123 +23,47 @@ import TableFooterPagination from '../common/TableFooterPagination'
-    })
-  }
-
-  handleChangePage(page) {
-    this.setState({ page })
-    const { fetchMore } = this.props.data
-    const { pageSize } = this.state
-
-    fetchMore({
-      variables: { page, pageSize },
-      updateQuery: (prev, { fetchMoreResult }) => fetchMoreResult
-    })
-  }
-
   render() {
-    if (this.props.data.loading) {
-      return (
-        <Paper style={{textAlign: 'center', padding: '5em'}}>
-          <Typography variant="display1" style={{color: "#e0e0e0"}}>
-            Loading...
-          </Typography>
-        </Paper>
-      )
-    }
-    const { auditTrails } = this.props.data
-    const { page, pageSize } = this.state
-
-    return (
-      <CardContent>
-        <Typography variant="headline" component="h3">
-          {this.props.title}
-        </Typography>
-        <Table>
-          <TableHead>
-            <TableRow>
-              <TableCell>User</TableCell>
-              <TableCell>Object</TableCell>
-              <TableCell>Action</TableCell>
-              <TableCell>Description</TableCell>
-              <TableCell>Time</TableCell>
-            </TableRow>
-          </TableHead>
-          <TableBody>
-            {auditTrails.entries.map(action => (
-              <TableRow key={action.id}>
-                <TableCell>{action.userEmail}</TableCell>
-                <TableCell>{action.object}</TableCell>
-                <TableCell>{action.action}</TableCell>
-                <TableCell>{action.description}</TableCell>
-                <TableCell>{formatDatetime(action.updatedAt)}</TableCell>
-              </TableRow>
-            ))}
-          </TableBody>
-          <TableFooterPagination
-            totalEntries={auditTrails.totalEntries}
-            page={page}
-            pageSize={pageSize}
-            handleChangePage={this.handleChangePage}
-            handleChangeRowsPerPage={this.handleChangeRowsPerPage}
-          />
-        </Table>
-      </CardContent>
-    )
-  }
-}
+    const { userId } = this.props
 
-const query = gql`
-  query AuditTrailsQuery ($userId: ID, $page: Int, $pageSize: Int) {
-    auditTrails (userId: $userId, page: $page, pageSize: $pageSize) {
-      entries {
-        id
-        userEmail
-        object
-        action
-        description
-        updatedAt
+    const columns = [
+      {
+        Header: 'User',
+        accessor: 'userEmail',
+        Cell: props => <span>{props.row.userEmail}</span>
       },
-      totalEntries,
-      totalPages,
-      pageSize,
-      pageNumber
-    }
-  }
-`
+      {
+        Header: 'Object',
+        accessor: 'object',
+        Cell: props => <span>{props.row.object}</span>
+      },
+      {
+        Header: 'Action',
+        accessor: 'action',
+        Cell: props => <span>{props.row.action}</span>
+      },
+      {
+        Header: 'Description',
+        accessor: 'description',
+        Cell: props => <span>{props.row.description}</span>
```

Comment:
for cases where you're just tossing the value in a `<span>`, you can just specify the `accessor:` and leave off the `Cell:`